cmd/minishift/cmd/start.go

```diff
@@ -267,11 +272,28 @@ func calculateDiskSizeInMB(humanReadableDiskSize string) int {
 	return int(diskSize / units.MB)
 }
 
+func determineIsoUrl(iso string) string {
+	iso = strings.ToLower(iso)
+
+	switch iso {
+	case b2dIsoAlias, "":
+		iso = constants.DefaultB2dIsoUrl
+	case centOsIsoAlias:
+		iso = constants.DefaultCentOsIsoUrl
+	default:
+		if !(strings.HasPrefix(iso, "https:") || strings.HasPrefix(iso, "http:") || strings.HasPrefix(iso, "file:")) {
```

Comment:
@hferentschik Here I think we are only validating the iso string a url from which it can be downloaded then better we should use `govalidator.IsURL(iso)` which will parse string and provide a bool if a URI or not. `"github.com/asaskevich/govalidator"` is already part of our dependencies.